### PR TITLE
User can drop media files, see them in the list, play them

### PR DESCRIPTION
**Pull request by lpender**

### CODE REVIEW

File: client/js/ingester.coffee [hunk 1]
```diff
@@ -5,18 +5,55 @@
...
+        ACL: "public-read",
+        Body: file,
+        Bucket: Meteor.settings.public.bucketName
+        ContentType: file.type,
+        Key: file.name,
+      }
+
+      bucket.putObject params, (err, data) ->
+        console.log(data)
+        resolve()
```
Comment (aaylward) [inline client/js/ingester.coffee:57]: should this reject if `err` is set?

File: client/js/ingester.coffee [hunk 2]
```diff
@@ -5,18 +5,54 @@
...
+
+  _uploadFile: (file) ->
+    return new Promise (resolve, reject) ->
+      bucket = new AWS.S3
+      params = {
+        ACL: "public-read",
+        Body: file,
+        Bucket: Meteor.settings.public.bucketName
+        ContentType: file.type,
+        Key: file.name,
```
Comment (aaylward) [inline client/js/ingester.coffee:53]: should the key identify the user? or will different users share the same actual file in s3?

Comment (aaylward) [inline client/js/ingester.coffee:53]: or should it be folder per user or something?

Comment (lpender) [inline client/js/ingester.coffee:53]: Probably a different folder per mix, and then yeah the filename should be a hash or something?

**Pull request merged**